### PR TITLE
Add map

### PR DESCRIPTION
**Pull request by Ingi-Hong**

### CODE REVIEW

**Comment by IanSaucy** [comment]
Preview URL: 
https://2e410a50.uncounteddeath.pages.dev/

File: src/styles/LandingPage.module.less [hunk 1]
```diff
@@ -6,23 +6,51 @@
...
+    margin: auto;
+    z-index: 100;
+    top: 7%;
+    right: 0;
+    left: 0;
+    width: 75.3%;
+    color: #373737;
+    display: inline-block;
+    font-size: calc(2.1vw + 26.1px); //arbitrary
     font-weight: 900;
```
Comment (IanSaucy) [inline src/styles/LandingPage.module.less:44]: Somehow the font in Zeplin looks "bolder". We might need to play around with the font-style or font-family more to get what we want.

File: src/pages/LandingPage.tsx [hunk 2]
```diff
@@ -22,10 +22,10 @@ export const LandingPage: React.FC = () => {
       <Space direction="vertical" size={150}>
         <Row>
           <Col>
-            <div className={styles.LandingPageFullWidthImagesContainer}>
-              <img src={landingPage1} className={styles.ImageStyle} />
+            <div className={styles.LandingPageMapImageContainer}>
+              <LandingPage1 className={styles.MapImageStyle}/>
               <p className={styles.LandingPageOverlayText}>
-                The Mortality Surveillance & Reporting Laboratory at Boston University
+                <strong>Millions</strong> of deaths from COVID-19 are underreported.
```
Comment (IanSaucy) [inline src/pages/LandingPage.tsx:28]: Could you move this into: https://github.com/UncountedDeaths/UncountedDeaths.github.io/blob/react_transfer/src/content/LandingPageContent.ts

You should be able to keep the `<strong>` tag in the string as well. Just name the variable something logical!

File: src/pages/LandingPage.tsx [hunk 3]
```diff
@@ -22,10 +22,10 @@ export const LandingPage: React.FC = () => {
       <Space direction="vertical" size={150}>
         <Row>
           <Col>
-            <div className={styles.LandingPageFullWidthImagesContainer}>
-              <img src={landingPage1} className={styles.ImageStyle} />
+            <div className={styles.LandingPageMapImageContainer}>
+              <LandingPage1 className={styles.MapImageStyle}/>
```
Comment (IanSaucy) [inline src/pages/LandingPage.tsx:26]: Good job on writing the CSS/LESS!

File: src/styles/LandingPage.module.less (hunk 1 above)
Comment (IanSaucy) [reply src/styles/LandingPage.module.less:44]: @minkyoseo Based on the preview you see do you think the text is "bold enough"?

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Great job! You did all the LESS stuff in a very logical way -- I am no expert in that regard but I understood what you did. 

Only want to @minkyoseo to verify the font boldness and have you move the "content" into the "content" folder as noted below. Otherwise good to go.

File: src/pages/LandingPage.tsx (hunk 2 above)
Comment (Ingi-Hong) [reply src/pages/LandingPage.tsx:28]: I tried just exporting the line as a string, but it reads the html tag as a string as well - was there another way you meant?

Comment (IanSaucy) [reply src/pages/LandingPage.tsx:28]: Oh, that's too bad. I'm not sure how to fix that issue then. Just leave it as is for now then!

File: src/styles/LandingPage.module.less (hunk 1 above)
Comment (IanSaucy) [reply src/styles/LandingPage.module.less:42]: Can you try switching the font size to REM and EM units? 

https://css-tricks.com/confused-rem-em/

Comment (IanSaucy) [reply src/styles/LandingPage.module.less:42]: Sizes are no `rem` in Zeplin for easy carryover.

Comment (Ingi-Hong) [reply src/styles/LandingPage.module.less:42]: Do we want to make the text size dynamic or is that planned for later down the line?

Comment (Ingi-Hong) [reply src/styles/LandingPage.module.less:42]: I switched font-size and line-height to rem

**Review by IanSaucy — APPROVED** [review]
lgtm

**Pull request merged**